src/Tasks/Microsoft.NET.Build.Tasks/build/Microsoft.NET.Publish.targets

```diff
@@ -501,7 +501,6 @@ Copyright (c) .NET Foundation. All rights reserved.
                                        RuntimeConfigPath="$(PublishRuntimeConfigFilePath)"
                                        RuntimeIdentifier="$(RuntimeIdentifier)"
                                        PlatformLibraryName="$(MicrosoftNETPlatformLibrary)"
-                                       RuntimeFrameworkVersion="$(RuntimeFrameworkVersion)"
```

Comment:
I'm not familiar with the internals of GenerateRuntimeConfigurationFiles task, but does removing this value mean it will instead use MicrosoftNETPlatformLibrary and project.assets.json to determine runtimeconfig version?